app/controllers/api/v1/moods_controller.rb

```diff
@@ -0,0 +1,40 @@
+class Api::V1::MoodsController < ApplicationController
+  before_action :get_user_id, only: [:index]
+  
+  def index
+    user_moods = Mood.user_moods(@user_id)
+    average_mood = Mood.average_mood(@user_id)
+
+    mood_summary = {
+      user_id: @user_id,
+      user_moods: user_moods,
+      avg_mood: average_mood
+    }
+    render json: MoodSerializer.new(mood_summary).serialize
+  end
+
+  def create
+    begin
+      mood = Mood.create!(mood_params)
+      head :created
+    rescue => e
```

Comment:
Should we look into abstracting this to the application controller for the error handling at a later, refactor, point? Def not necessary, but might be nice to slim down the controller a little bit!

Reply:
Oh this is easy I can do this before merging